### PR TITLE
skip pipelineresource tests

### PR DESCRIPTION
**Pull request by Basavaraju-G**
# Changes
pipelineResource is been deprecated in latest release of pipeline. skip those tests while running cli e2e tests and add new plugin related tests. https://github.com/tektoncd/pipeline/pull/6150.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by Basavaraju-G** [comment]
Hi @afrittoli, @bobcatfish  would you please review this PR https://github.com/tektoncd/plumbing/pull/1392

**Comment by wlynch** [comment]
@tektoncd/cli-maintainers - can someone take a look? 🙏 

FWIW CLI may want to keep some support around for pipeline resources - users were still able to create them until v0.46.0 which means that they can still be created in [Pipeline LTS versions v0.41.0 and v0.44.0](https://github.com/tektoncd/pipeline/blob/main/releases.md#release).

**Comment by piyush-garg** [comment]
> @tektoncd/cli-maintainers - can someone take a look? pray
> 
> FWIW CLI may want to keep some support around for pipeline resources - users were still able to create them until v0.46.0 which means that they can still be created in [Pipeline LTS versions v0.41.0 and v0.44.0](https://github.com/tektoncd/pipeline/blob/main/releases.md#release).

@wlynch there is CLI 0.30.0 LTS release which will have the support for pipeline resources, in 0.31.0 release we are bumping the pipeline vendor to 0.46 and hence we are removing the pipelineresources. Let me know if we need to do something else. cc @vdemeester @vinamra28 @prataap007

**Comment by wlynch** [comment]
/uncc @wlynch

I'm happy to defer to a review from @tektoncd/cli-maintainers here - feel free to assign me for submission once that's done!
/cc @piyush-garg do you want to review this? 👀

**Review by vinamra28 — APPROVED** [review]
/lgtm

**Review by vdemeester — APPROVED** [review]

**Pull request merged**
